.github/workflows/check-in-tree-build.yml

```diff
@@ -61,6 +61,9 @@ jobs:
             clang-${{ env.LLVM_VERSION }} \
             spirv-tools
           sudo update-alternatives --install /usr/bin/clang clang /usr/bin/clang-${{ env.LLVM_VERSION }} 1000
+          # Ensure that the pre-installed Clang path gets detected by the
+          # in-tree-built llvm-lit tool
+          export CLANG=/usr/bin/clang
```

Comment:
If I understand correctly, it only applies to the particular step, because each step is launched in its separate shell.

If you are looking for global env variables see `env` section above. I also suggest inspecting/debugging `llvm-project/llvm/utils/lit/lit/llvm/config.py:428:` to see why we failed to find clang in the first place

Reply:
> It only applies to the particular step, because each step is launched in its separate shell.

Thanks, I was struggling to interpret the failures myself.

> I also suggest inspecting/debugging llvm-project/llvm/utils/lit/lit/llvm/config.py:428: to see why we failed to find clang in the first place.

In progress; I was intending to do this anyway before converting the PR from draft.